libyui-rake.gemspec

```diff
@@ -17,31 +19,36 @@
 Gem::Specification.new do |spec|
   # gem name and description
   spec.name	= "libyui-rake"
-  spec.version	= File.read(File.expand_path("../VERSION", __FILE__)).chomp
+  spec.version	= File.read(File.expand_path("VERSION", __dir__)).chomp
   spec.summary	= "Rake tasks that provide basic workflow for libyui development"
   spec.license = "LGPL-2.1"
 
   # author
   spec.author	= "YaST team"
-  spec.email	= "yast-devel@suse.com"
-  spec.homepage	= "https://github.com/openSUSE/libyui-rake"
-
-  spec.summary = "Rake tasks providing basic workflow for libyui development"
-  spec.description = <<-end
-Rake tasks that support the workflow of a libyui developer. It allows packaging
-a repo, sending it to the build service, creating a submit request to the
-target repo or running the client from the git repo.
-Heavily inspired by yast-rake.
-end
+  spec.email	= "yast-devel@lists.opensuse.org"
+  spec.homepage	= "https://github.com/libyui/libyui-rake"
+
+  spec.description = <<~DESCRIPTION
+    Rake tasks that support the workflow of a libyui developer. It allows packaging
+    a repo, sending it to the build service, creating a submit request to the
+    target repo or running the client from the git repo.
+    Heavily inspired by yast-rake.
+  DESCRIPTION
 
   # gem content
   spec.files = Dir["lib/**/*.rb", "lib/tasks/*.rake", "data/*", "COPYING", "README.md", "VERSION"]
 
   # define LOAD_PATH
   spec.require_path = "lib"
 
+  # Ruby 2.5.0+ is required
+  spec.required_ruby_version = ">= 2.5.0"
+
   # dependencies
-  spec.add_runtime_dependency("rake", "> 10.0", "< 99")
-  spec.add_runtime_dependency("packaging_rake_tasks", "~> 1.5")
   # "~> 1.5"  means ">= 1.5 and < 2" (thanks to darix)
+  spec.add_runtime_dependency("packaging_rake_tasks", "~> 1.5")
+  spec.add_runtime_dependency("rake", "> 10.0", "< 99")
+  # enable MFA (2FA) authentication at rubygems.org
+  # see https://guides.rubygems.org/mfa-requirement-opt-in/
+  spec.metadata["rubygems_mfa_required"] = "true"
```

Comment:
This was automatically added by Rubocop, but IMHO it makes sense. See https://guides.rubygems.org/mfa-requirement-opt-in/ for more details.

Personally I have enabled MFA. @jreidinger @mvidner @imobachgs, what about you? Is this change OK?

Reply:
well, I would have to enable it, but not critical for me.

Reply:
It is OK for me.

Reply:
Well, I haven't tried this option yet. Can we lock ourselves out if the other maintainers haven't either? :rofl:

Reply:
You cannot lock out yourselves, if you do not use MFA then just enable it and then you can do whatever you need.

Reply:
fine for me